awscli/customizations/scalarparse.py

```diff
@@ -36,13 +40,40 @@ def identity(x):
     return x
 
 
+def iso_format(value):
+    return parse_timestamp(value).isoformat()
+
+
+def add_timestamp_parser(session):
+    factory = session.get_component('response_parser_factory')
+    # If a --profile is provided that does not exist, loading
+    # a value from get_scoped_config will crash the CLI.
+    # This function can be called as the first handler for
+    # the session-initialized event, which happens before a
+    # profile can be created, even if the command would have
+    # successfully created a profile. Instead of crashing here
+    # on a ProfileNotFound the CLI should just use 'none'.
+    try:
+        timestamp_format = session.get_scoped_config().get(
+            'cli_timestamp_format',
+            'none')
+    except ProfileNotFound:
```

Comment:
Nevermind, I see the comment now. Might be good to move the comment to under the exception as I completely missed reading that comment